src/components/playlist-countdown/playlist-countdown.js

```diff
@@ -161,6 +167,7 @@ class PlaylistCountdown extends Component {
     const className = [style.playlistCountdown];
     const isHidden = !this.state.timeToShow || countdown.duration >= props.duration;
     const isCanceled = this.props.countdownCanceled;
+    this.setState({isHiddenOrCanceled: isHidden || isCanceled});
```

Comment:
I would change the state to `show` (invert the condition for sure)

Reply:
fixed